tools/depends/target/ffmpeg/CMakeLists.txt

```diff
@@ -56,12 +56,9 @@ elseif(CORE_SYSTEM_NAME STREQUAL osx)
                           --disable-decoder=mpeg_xvmc --disable-vda --disable-crystalhd --enable-videotoolbox
                           --target-os=darwin
                           --disable-securetransport)
-elseif(CORE_SYSTEM_NAME STREQUAL rbpi)
-  list(APPEND ffmpeg_conf --cpu=${CPU} --disable-vaapi --disable-vdpau)
-endif()
 
-if(CPU MATCHES arm OR CORE_SYSTEM_NAME STREQUAL rbpi)
-  list(APPEND ffmpeg_conf --enable-pic --disable-armv5te --disable-armv6t2)
+if(CPU MATCHES arm)
+  list(APPEND ffmpeg_conf --enable-pic --disable-armv5te --disable-armv6t2 --disable-vaapi --disable-vdpau)
```

Comment:
this is wrong and will lead to problems in the future. why should an arm cpu not be able to support vdpau?